index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en-us">
+<head>
+  <meta charset="utf-8">
+  <link rel="stylesheet" type="text/css" href="styles.css">
+  <title>SendGrid - Customers</title>
+</head>
+<body>
+  <div class="container">
```

Comment:
Use semantic elements when possible, this could be a `section` element.
https://developer.mozilla.org/en-US/docs/Glossary/Semantics